### DISCUSSION TITLE
XMake / XRepo support

### DISCUSSION THREAD

**Discussion by Arthapz** [ned14/llfio#104] [Show and tell]
Hi,
Just want to say that i added the library to XRepo (the XMake Dependency repos) :D 

https://github.com/xmake-io/xmake-repo/pull/1631

```
xrepo install llfio
```

(to make it work, i ported the build system to XMake, as the CMake was a bit complicated)

**Comment by ned14** [comment]
Thanks for the contribution.

Did I see some patching of the sources in there? A PR so they could go into trunk would be nice.
